euth/dashboard/forms.py

```diff
@@ -114,7 +116,11 @@ class ProjectForm(forms.ModelForm):
     class Meta:
         model = project_models.Project
         fields = ['name', 'description', 'image', 'information', 'is_public',
-                  'result']
+        'result']
+        widgets = {'is_public': RadioSelect(choices=[
+            (True, 'Private'),
```

Comment:
Good work @ahmedjenbaz :smile: The translatable version ` _('Private')` might be better here.